src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: function (num) {
+      return this.result + num;
+    },
+    subtract: function (num) {
+      return this.result - num;
+    },
+    multiply: function (num) {
+      return this.result * num;
+    },
+    divide: function (num) {
+      return this.result / num;
+    },
```

Comment:
Instead of using the `function` keyword, you can use method shorthand in object literals. This is a code style issue that makes the code cleaner and more concise.

BAD EXAMPLE:
 add: function (num) {
  return this.result + num;
},

GOOD EXAMPLE:
 add(num) {
  this.result += num;
},

Please update the methods `add`, `subtract`, `multiply`, and `divide` to use this shorthand syntax.